src/Compilers/VisualBasic/Portable/Binding/InitializerSemanticModel.vb

```diff
@@ -133,7 +133,8 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
                     Throw ExceptionUtilities.UnexpectedValue(Me.MemberSymbol.Kind)
             End Select
 
-            Return boundInitializer
+            Dim expressionInitializer = TryCast(boundInitializer, BoundExpression)
+            Return If(expressionInitializer IsNot Nothing AndAlso expressionInitializer.Kind <> BoundKind.BadExpression, New BoundEqualsValue(initializer, expressionInitializer), boundInitializer)
```

Comment:
Producing proxy nodes for all initializers (including those that aren't expressions) caused numerous test failures--some in flow analysis tests and some in various editor tests. Wrapping expression cases is sufficient for extracting IOperations, and don't lead to test failures in other areas.